ssl/quic/quic_stream_map.c

```diff
@@ -757,7 +757,8 @@ void ossl_quic_stream_map_remove_from_accept_queue(QUIC_STREAM_MAP *qsm,
     --qsm->num_accept;
 
     if ((max_streams_rxfc = qsm_get_max_streams_rxfc(qsm, s)) != NULL)
-        ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt);
+        if(!ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt))
+            return;
```

Comment:
The change is not changing anything. There is no point in checking the returned value as the function will return anyway and it is a void function. I'd recommend casting the function to `(void)` instead of checking the return value to indicate that the not checked return is not an inadvertent omission.